### ISSUE TITLE
Choose requirements for each level of participation in Legalese

### ISSUE THREAD

**Issue by virgil** [legalese-io/legalese-io.github.io#229]
Per: https://legalese.slack.com/archives/strategy/p1476076745000130
1. Public groups like talk and GitHub 
2. Slack 
3. Collective 
4. Google drive

**Comment by virgil** [comment]
For Drive access I propose it be signing a NDA.

**Comment by wabisabiyo** [comment]
Virgil why would you want the stricter / more restrictive NDA? If anything, I think it should be a CLA instead. Which means... we need to restart conversation on licenses.

**Comment by virgil** [comment]
Yes you're right.  NDA too much.  I think I even thought of CLA but I said NDA out of habit.
